content/en/docs/instrumentation/net/automatic/_index.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Automatic Instrumentation
+description: Send traces and metrics from .NET applications and services.
+linkTitle: Automatic
+cSpell:ignore: coreutils HKLM iisreset myapp
+weight: 20
+---
+
+Use the OpenTelemetry .NET Automatic Instrumentation to send traces and metrics
+from .NET applications and services to observability back ends without having to
+modify their source code.
+
+To learn how to manually instrument your service or app code, see
+[Manual instrumentation](../manual).
+
+## Compatibility
+
+OpenTelemetry .NET Automatic Instrumentation should work with all officially
+supported operating systems and versions of
+[.NET](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core).
+
+The minimal supported version of
+[.NET Framework](https://dotnet.microsoft.com/download/dotnet-framework) is
+`4.6.2`.
+
+CI tests run against the following operating systems:
+
+- [Alpine](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docker/alpine.dockerfile)
+- [CentOS 7](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docker/centos-build.dockerfile)
+- [macOS Big Sur 11](https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md)
+- [Microsoft Windows Server 2022](https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md)
+- [Ubuntu 20.04 LTS](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md)
+
+{{% alert title="Note" color="note" %}} ARM architectures are not supported. See
+[#2181](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/2181)
+for more information. {{% /alert %}}
+
+## Setup
+
+To instrument a .NET application automatically, download and run the installer
+script for your operating system.
+
+### Linux and macOS
+
+Download and run the `.sh` script:
+
+```shell
+# Download the bash script
+curl -sSfL https://raw.githubusercontent.com/open-telemetry/opentelemetry-dotnet-instrumentation/v1.0.0-rc.2/otel-dotnet-auto-install.sh -O
+
+# Install core files
+sh ./otel-dotnet-auto-install.sh
+
+# Enable execution for the instrumentation script
+chmod +x $HOME/.otel-dotnet-auto/instrument.sh
+
+# Setup the instrumentation for the current shell session
+. $HOME/.otel-dotnet-auto/instrument.sh
+
+# Run your application with instrumentation
+OTEL_SERVICE_NAME=myapp OTEL_RESOURCE_ATTRIBUTES=deployment.environment=staging,service.version=1.0.0 ./MyNetApp
+```
+
+{{% alert title="Note" color="note" %}} On macOS
+[`coreutils`](https://formulae.brew.sh/formula/coreutils) is required.
```

Comment:
~~~suggestion
[`coreutils`](https://formulae.brew.sh/formula/coreutils) is required. If you have [homebrew](https://brew.sh/) installed, you can simply get it by running

```
brew install coreutils
```
~~~